### PR TITLE
feat(assistant): Update conversation header

### PR DESCRIPTION
**Pull request by knajjars**
- Updated: IconButton to be a named export for consistency
- Updated: Conversation Header to match new designs
- Pending: Add About/Edit and Share items
- Fixed: The `c/[:id]` was conflicting with the query `?id=`, which will be used for identifying `assistants`. For which the new query value is `?assistantId=`

https://github.com/cohere-ai/cohere-toolkit/assets/19353218/5fe1b3c6-c466-4971-8921-232dc6b55b44

**AI Description**

This pull request updates the import statements for the `IconButton` component in multiple files. The changes involve adding curly braces `{}` around the component name in the import statement, changing it from `import IconButton from '@/components/IconButton';` to `import { IconButton } from '@/components/IconButton';`. This modification ensures that only the `IconButton` component is imported, rather than any other named exports from the same file.

- **Files affected:**
  - `src/interfaces/coral_web/src/components/Agents/AgentCard.tsx`
  - `src/interfaces/coral_web/src/components/Agents/AgentsSidePanel.tsx`
  - `src/interfaces/coral_web/src/components/Agents/DiscoverAgentCard.tsx`
  - `src/interfaces/coral_web/src/components/Citations/Citation.tsx`
  - `src/interfaces/coral_web/src/components/Citations/CitationDocumentHeader.tsx`
  - `src/interfaces/coral_web/src/components/Conversation/ComposerMenu.tsx`
  - `src/interfaces/coral_web/src/components/Conversation/ConfigurationDrawer.tsx`
  - `src/interfaces/coral_web/src/components/Conversation/Header.tsx`
  - `src/interfaces/coral_web/src/components/ConversationList/ConversationListHeader.tsx`
  - `src/interfaces/coral_web/src/components/MessageRow.tsx`

- **Code changes:**
  - In all the affected files, the import statement for `IconButton` is updated as follows:

```jsx
- import IconButton from '@/components/IconButton';
+ import { IconButton } from '@/components/IconButton';
```

### CODE REVIEW

File: src/interfaces/coral_web/src/components/Shared/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -142,11 +142,11 @@ export const Tooltip: React.FC<TooltipProps> = ({
             {...getFloatingProps()}
           >
             {size === 'sm' ? (
-              <Text styleAs="p-sm" className="text-marble-300">
+              <Text as="span" styleAs="p-sm" className="text-marble-300">
```
Comment (wujessica) [inline src/interfaces/coral_web/src/components/Shared/Tooltip/Tooltip.tsx:145]: don't ask me why but this fixes the tooltip being oddly tall
```suggestion
              <Text as="span" styleAs="p-sm" className="flex text-marble-300">
```

|after|before|
|-|-|
|image|image|

File: src/interfaces/coral_web/src/components/Conversation/Header.tsx [hunk 2]
```diff
@@ -132,20 +116,30 @@ export const Header: React.FC<Props> = ({ conversationId, isStreaming }) => {
...
         <span className="flex items-center gap-x-2 py-4 pl-4 md:pl-0">
-          <KebabMenu
-            className={cn('md:hidden', { hidden: !conversationId })}
-            items={menuItems}
-            anchor="left start"
-          />
+          <KebabMenu className="md:hidden" items={menuItems} anchor="left start" />
           <IconButton
-            iconName="trash"
-            onClick={handleDelete}
```
Comment (wujessica) [inline src/interfaces/coral_web/src/components/Conversation/Header.tsx:142]: can we no longer delete chats? :')

**Review by wujessica — APPROVED** [review]

File: src/interfaces/coral_web/src/components/Conversation/Header.tsx (hunk 2 above)
Comment (knajjars) [reply src/interfaces/coral_web/src/components/Conversation/Header.tsx:149]: it was removed from designs, looks like only possible from conversation list panel ;(

File: src/interfaces/coral_web/src/components/IconButton.tsx
[the remark this answers is not in the document]
Comment (knajjars) [reply src/interfaces/coral_web/src/components/IconButton.tsx:1]: done!

**Pull request merged**